pkg/serviceaccounts/controllers/create_dockercfg_secrets.go

```diff
@@ -37,14 +41,16 @@ type DockercfgControllerOptions struct {
 	// If zero, re-list will be delayed as long as possible
 	Resync time.Duration
 
-	DefaultDockerURL string
+	// DockerURLsIntialized is used to send a signal to this controller that it has the correct set of docker urls
```

Comment:
Can you add the source of the signal (DockerRegistryServiceController) to the doc?